roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds.yaml.j2

```diff
@@ -33,9 +33,9 @@ spec:
               mountPath: "/etc/kubernetes/node-feature-discovery/source.d/"
 
         - name: kubevirt-cpu-nfd-plugin
-          image: {{ ssp_registry | default('quay.io/ksimon') }}/{{ kubevirt_cpu_nfd_plugin_image }}:{{ kubevirt_cpu_nfd_plugin_tag }}
+          image: {{ ssp_registry | default('quay.io/kubevirt') }}/{{ kubevirt_cpu_nfd_plugin_image }}:{{ kubevirt_cpu_nfd_plugin_tag }}
```

Comment:
Since the registry is now the same for everything, what about setting ssp_registry in the role defaults yaml and keeping just `{{ ssp_registry }}` in the code?

Reply:
virt-launcher image uses different registry (dockerhub).

Reply:
I see. Well then we have to keep it like it is.